JINSU/Domino/Assets/CodeScript/ChasingCamera.cs

```diff
@@ -0,0 +1,62 @@
+using System.Resources;
+using UnityEngine;
+
+
+public class ChasingCamera : MonoBehaviour
+{
+
+    // Start is called once before the first execution of Update after the MonoBehaviour is created
+    void Start()
+    {
+       
+    }
+
+    //����� ����ȭ���� �Է�ó���ؼ� �����ϰ� �����ִ� ī�޶� �׽�Ʈ
+
+    // Update is called once per frame
+    void Update()
+    {
+        MoveCamera();
+
+    }
+
+// for testing
+    private void MoveCamera()
+    {
+        float moveZ = 0f;
+        float moveX = 0f;
+        if (Input.GetKey(KeyCode.W))
+        {
+            moveZ += 1f;
+        }
+
+        if (Input.GetKey(KeyCode.S))
+        {
+            moveZ -= 1f;
+        }
+
+        if (Input.GetKey(KeyCode.A))
+        {
+            moveX -= 1f;
+        }
+
+        if (Input.GetKey(KeyCode.D))
+        {
+            moveX += 1f;
+        }
+
+        transform.Translate(new Vector3(moveX, 0f, moveZ) * 0.1f);
```

Comment:
직접 wsad에 대해서 if문을 처리하기 보다, 유니티의 Input Manager를 사용하는 방식으로 변경해주면 더 간결하게 작성할 수 있을 다고 하네요
https://docs.unity3d.com/kr/2021.3/Manual/class-InputManager.html
```suggestion
      float moveSpeed = 10;
//Define the speed at which the object moves.

float horizontalInput = Input.GetAxis("Horizontal");
//Get the value of the Horizontal input axis.

float verticalInput = Input.GetAxis("Vertical");
//Get the value of the Vertical input axis.

transform.Translate(new Vector3(horizontalInput, verticalInput, 0) * moveSpeed * Time.deltaTime);
//Move the object to XYZ coordinates defined as horizontalInput, 0, and verticalInput respectively.
```

Reply:
다음주에 작업해보도록 하겠습니다. ✌️